### ISSUE TITLE
Flare

### ISSUE THREAD

**Issue by LukyLucas** [MineInAbyss/Issue-Tracker#118]
### Is your feature request related to a problem?

I think this would be good for the adventure mode however the basic idea is theres too different types, help flare and light flare.

help flare- gives players glowing effect for 3 minutes allowing nearby players to quickly locate them. double edged sword if its a hostile player or peaceful player / teammate.

light flare- not very useful if they have Fullbright but the basic idea is when used it creates a ball of light in the air and lights up a large area for a limited time then fades.

the flares would either be fired from crossbows or fired by hand so the base could probably just be fireworks with extra code.
as an extra feature to it, the flares would have a 1 in 5 chance of spawning a random creature from that layer.

### Describe the solution you'd like.

for the flare firework effects they would both have strong trails on their way up however the help one would have a strong red burst effect and the light one would have a white small ball effect. the help one is pretty basic where you fire it and it goes up explodes and you get the glowing effect. the light one would need some extra work but i think it can be done by replacing air blocks with light blocks. it would probably need several of these each with a larger radius so its weaker the farther from the source. 
as a additional graphic the light flares small ball effect could stay in the air and get smaller till the light dissipates.

### Describe alternatives you've considered.

i thought this was a cool idea but im sure there will be some people who are like, "this is unnecessary don't bother"
however, i just want to say that these would truly be great additions that would maybe be widely used in adventure and i could see them being used to locate friends or light areas while building in survival.

### Agreements

- [X] I have searched for and ensured there isn't already an open issue regarding this.
- [X] I have ensured the feature I'm requesting isn't already in the latest supported Paper build.

### Other

if this feature is given the greenlight im willing to do specific textures for them if needed. lemme know if there's suggestions for crafting recipes.

**Comment by Kasaichan** [comment]
Could be interpreted as making fireworks more accessible.
![image](https://user-images.githubusercontent.com/102488431/164781235-54068f59-702c-4f64-9b2e-90705dd53bb1.png)

**Comment by LukyLucas** [comment]
Good point... Maybe it can have damage off or only work when fired upwards. I don't want it to be used for easy access to rockets. I want it to be more of a player support item.
